roads.mss

```diff
@@ -3100,3 +3100,87 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
     }
   }
 }
+#railways-text-name {
+  /* Render railway=rail from z11 */
+  [railway = 'rail'][zoom >= 11] {
+    text-name: "[name]";
+    text-fill: #333333;
+    text-size: 8;
```

Comment:
We're trying to get rid of too small text sizes, see https://github.com/gravitystorm/openstreetmap-carto/issues/2209.